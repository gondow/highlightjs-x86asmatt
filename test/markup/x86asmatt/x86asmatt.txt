# add5.s
    .text
    .globl add5
    .type add5, @function
add5:
    pushq %rbp
    movq  %rsp, %rbp
    movl  %edi, -4(%rbp)
    movl  -4(%rbp), %eax
    addl  $5, %eax
    popq  %rbp
    ret
    .size  add5, .-add5
