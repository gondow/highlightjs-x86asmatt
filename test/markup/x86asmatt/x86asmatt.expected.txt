<span class="hljs-comment"># add5.s</span>
    <span class="hljs-built_in">.text</span>
    <span class="hljs-built_in">.globl</span> add5
    <span class="hljs-built_in">.type</span> add5, <span class="hljs-name">@function</span>
<span class="hljs-string">add5:</span>
    <span class="hljs-keyword">pushq</span> <span class="hljs-title">%rbp</span>
    <span class="hljs-keyword">movq</span>  <span class="hljs-title">%rsp</span>, <span class="hljs-title">%rbp</span>
    <span class="hljs-keyword">movl</span>  <span class="hljs-title">%edi</span>,<span class="hljs-name"> -4</span>(<span class="hljs-title">%rbp</span>)
    <span class="hljs-keyword">movl</span> <span class="hljs-name"> -4</span>(<span class="hljs-title">%rbp</span>), <span class="hljs-title">%eax</span>
    <span class="hljs-keyword">addl</span>  <span class="hljs-name">$5</span>, <span class="hljs-title">%eax</span>
    <span class="hljs-keyword">popq</span>  <span class="hljs-title">%rbp</span>
    <span class="hljs-keyword">ret</span>
    <span class="hljs-built_in">.size</span>  add5, .-add5
